test/synth/synth-stdout.test.ts

```diff
@@ -1,13 +1,19 @@
+import { readdirSync } from 'fs';
 import * as path from 'path';
 import * as fs from 'fs-extra';
 import * as yaml from 'yaml';
-import { Config, ValidationConfig } from '../../src/config';
-import { findConstructMetadata, mkdtemp } from '../../src/util';
+import { Config, HelmChartApiVersion, SynthesisFormat, ValidationConfig } from '../../src/config';
+import { crdsArePresent, findConstructMetadata, hashAndEncode, mkdtemp } from '../../src/util';
+
+const DEFAULT_APP = 'node index.js';
+const imports: string[] = [];
 
 beforeEach(() => {
   // resetting so that every test can use a different config file,
   // which is read on module load.
   jest.resetModules();
+  // Emptying the imports every time a test is run
+  imports.length = 0;
```

Comment:
This is highly irregular and is normally a code smell. Lets talk about it online.